README.md

```diff
@@ -373,17 +356,24 @@ All images are pushed to [Docker Hub](docker_hub_tags).
 
 Fixed version tags (`x.y.z`) are frozen: we never update them.
 
+Example of minor versions timeline:
+
+- `x.y.0` - Alpha
+  - `x.y.1` - Beta
+    - `x.y.2` - Stable
+      - `x.y.3` - Stable
+
 Rolling versions are updated to the last fixed version tags:
 
 - `x.y` == `x.y.<last-z>` (`==` means 'points to the same image')
-- `1` == `1.<last-y>.<last-z>`
-- `2` == `2.<last-y>.<last-z>`
+- `1` == `1.<last-y>.2`
+- `2` == `2.<last-y>.2`
 - `latest` == `2`
 
 Special builds (CentOS) are updated with the same policy as rolling versions:
 
-- `1.x-centos7` image offers a last `1.<last-y>.<last-z>` release
-- `2.x-centos7` image offers a last `2.<last-y>.<last-z>` release
+- `1.x-centos7` image offers a last `1.<last-y>.2` release
```

Comment:
This is confusing as a hell. But after 2nd thought I understood it!

Reply:
Any proposal? Or it'll out of scope here?

Reply:
Nope. No proposals :)

P.S.
When you see magical `.2` it confuses a lot...